src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -48,8 +49,15 @@
   public CompilerOptions getCompilerOptions() {
     final CompilerOptions options = new CompilerOptions();
     options.setClosurePass(true);
+
+    // All diagnostics are WARNINGs (or off) and thus ignored unless debug == true.
+    // Only report issues (and fail for them) that are specifically causing problems for Clutz.
+    // The idea is to not do a general sanity check of Closure code, just make sure Clutz works.
+    // Report missing types as errors.
```

Comment:
Sounds like we are becoming more permissive, allowing wrong Closure code. Wouldn't those errors sooner or later end up in a js_binary rule and break the compilation?